printer_mods/Koios/6mm_Glass/README.md

```diff
@@ -0,0 +1,10 @@
+# Glass mount
+
+This is a modification of the front panel locks from 2.1 to support holding 6mm tempered glass panels.  I used 3m VHB to adhere them to my glass panels so that thickness was considered as well.  The tab which supports the glass was thickened and has proven to be very robust on my printer. This can be run with 2.4 without issue, using panel sizes from the 2.1 bom.
+
+I've also added 'glass rest.stl' & the step file to secure the side panels with quick releases as well.  This is necessary as the glass is quite heavy and the lock levers wont hold it from sliding.
+
```

Comment:
Are the overhangs on the bottom of `glass_rest.STL` more than 45 degrees?  It looks like it's just a filet, which would make the overhangs steeper than 45 degrees, and thus could be problematic to print.  Maybe they should just have 45 degree chamfers.
<img width="255" alt="Screen Shot 2021-03-21 at 10 19 48" src="https://user-images.githubusercontent.com/69389593/111908343-40c9e000-8a2f-11eb-8010-c8e45d507346.png">

Reply:
Good catch. I'm surprised how well it came out with this.  I've updated the stl / step, as well as the other tweaks in readme you requested.